Notify-Tech-Docs/source/documentation/python.md

```diff
@@ -18,49 +20,87 @@ from notifications_python_client.notifications import NotificationsAPIClient
 notifications_client = NotificationsAPIClient(api_key)
 ```
 
-To get an API key, [log in to GOV.UK Notify](https://www.notifications.service.gov.uk/) and go to the _API integration_ page. Information on API keys can be found [here](/#api-keys).
+To get an API key, [log in to GOV.UK Notify](https://www.notifications.service.gov.uk/) and go to the _API integration_ page. More information can be found in the [API keys](/#api-keys) section.
 
 # Send a message
 
+GOV.UK Notify enables you to send text messages, emails and letters.
```

Comment:
Pretty sure _enables_ is against GOV.UK style. I’d go with something like ‘You can use GOV.UK Notify to send text messages, emails and letters.’